### PR TITLE
[Reference cleanup]LPS-163319

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references Misc. Thanks!
https://issues.liferay.com/browse/LPS-163319

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @catarineaguiar , Could you please help to scan the test results? thx

**Comment by catarineaguiar** [comment]
ci:test:relevant

File: modules/apps/blogs/blogs-demo/src/main/java/com/liferay/blogs/demo/internal/BlogsDemo.java [hunk 1]
```diff
@@ -139,11 +125,14 @@ private <T> T _getRandomElement(List<T> list) {
 
 	private final List<BlogsEntryDemoDataCreator> _blogsEntryDemoDataCreators =
```
Comment (dantewang) [inline modules/apps/blogs/blogs-demo/src/main/java/com/liferay/blogs/demo/internal/BlogsDemo.java:126]: @JanisZhang It's weird that we reference fixed number of `BlogsEntryDemoDataCreator` services, put them into a list, and use them by iterating the list. Currently there are only 2 of them and both are referenced statically, so let's remove the list field, and call these `BlogsEntryDemoDataCreator` instances directly.

If in the future they want more to be added, they should use something like service tracker list.

**Review by dantewang — CHANGES_REQUESTED** [review]
@JanisZhang Please add an SF commit based on my comment, thanks!

**Comment by Kvale1733** [comment]
@JanisZhang Regression Found

1. This error appears on startup for all tests:
`2022-09-19 20:55:16.764 WARN  [Start Level: Equinox Container: eda00dfb-cfd0-456f-aeb7-6cdaf1fa4615][LDAPConfigurationListener:123] Unable to register configurations
[beanshell] java.lang.NullPointerException: null
[beanshell] 	at com.liferay.portal.security.ldap.internal.configuration.LDAPConfigurationListener.setConfigurationProvider(LDAPConfigurationListener.java:112) [bundleFile:?]
[beanshell] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]`[Testray](https://testray.liferay.com/home/-/testray/case_results/1983125064?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1983124963%26testrayRunId%3D1983125038%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

**Comment by JanisZhang** [comment]
ci:close

**Pull request closed without merging**